scripts/portal.js

```diff
@@ -16,8 +16,10 @@ function Portal (sites) {
   }
 
   function _directory (sites) {
+    const siteTypesArray = [...new Set(sites.map(site => site.type).filter(Boolean))]
+    const siteTypes = siteTypesArray.reduce((output, siteType) => `${output}<a data-type='${siteType}' href='#' onclick='event.preventDefault(); toggleType(event)'>&lt;${siteType}&gt;</a>`, `<a class='current' href='#' onclick='event.preventDefault(); toggleType(event)'>&lt;all&gt;</a>`)
```

Comment:
I don't believe the `event.preventDefault()` is actually doing anything here is it?

Reply:
it prevents the `#` from href to apply to url

Reply:
Ahhh, duh, of course. Thanks!